coderd/database/queries/templates.sql

```diff
@@ -22,17 +22,23 @@ WHERE
 			organization_id = @organization_id
 		ELSE true
 	END
+	-- Filter by organization_name
+	AND CASE
+		WHEN @organization_name :: text != '' THEN
+			LOWER("organization_name") = LOWER(@organization_name)
+		ELSE true
+	END
```

Comment:
This needs to be removed.

The param on the frontend is `organization` and it accepts both an ID or an org name.

https://github.com/coder/coder/blob/604d5cb552449abbd9b650069a0e3bd6df9292b5/coderd/searchquery/search.go#L213-L227